example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,33 +1,22 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.radar.example">
   <uses-permission android:name="android.permission.INTERNET"/>
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
   <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
-  <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
```

Comment:
@gabrielalao some of these permissions we may not need.  But we do need things related to location, bluetooth and network.

Reply:
@tjulien gotcha, I will check permission again